tidb-cloud/troubleshoot-import-access-denied-error.md

```diff
@@ -117,35 +121,37 @@ In the sample trust entity:
+1. In the AWS Management Console, open the Amazon S3 console, go to the **Buckets** page. A list of buckets are displayed.
+2. Select and click on the target bucket and the bucket information page is displayed.
+3. On the bucket information page, scroll down to find the **Object Ownership** area. Make sure that the "Object Ownership" configuration is "Bucket owner enforced".
 
-If the configuration is not "Bucket owner enforced", the `AccessDenied` error occurs, because your account does not have enough permissions for all objects in this bucket.
+    If the configuration is not "Bucket owner enforced", the `AccessDenied` error occurs, because your account does not have enough permissions for all objects in this bucket.
 
 To handle the error, click **Edit** in the upper-right corner of the Object Ownership area and change the ownership to "Bucket owner enforced". Note that this might affect your other applications that are using this bucket.
 
 ## Check your bucket encryption type
 
 There are more than one way to encrypt an S3 bucket. When you try to access the objects in a bucket, the role you have created must have the permission to access the encryption key for data decryption. Otherwise, the `AccessDenied` error occurs.
 
-To check the encryption type of your bucket, open the Amazon S3 console in the AWS Management console, choose the name of the target bucket, choose **Properties**, and you will see the **Default encryption** page that shows the encryption type of the bucket.
+To check the encryption type of your bucket, open the Amazon S3 console in the AWS Management console and click **Buckets** on the left navigation bar, choose the name of the target bucket, choose **Properties**, and you will see the **Default encryption** area that shows the encryption type of the bucket.
 
 There are two types of server-side encryption: Amazon S3-managed key (SSE-S3) and AWS Key Management Service (SSE-KMS). For SSE-S3, further check is not needed because this encryption type does not cause access denied errors. For SSE-KMS, you need to check the following:
 
-- If the AWS KMS key ARN on the page is displayed in black without an underline, the AWS KMS key is an AWS-managed key (aws/s3).
-- If the AWS KMS key ARN on the page is displayed in blue with an underline, click the key ARN to see the specific encryption type. It might be an AWS managed key (aws/s3) or a customer-managed key.
+- If the AWS KMS key ARN in the area is displayed in black without an underline, the AWS KMS key is an AWS-managed key (aws/s3).
+- If the AWS KMS key ARN in the area is displayed in blue with an underline, click on the key ARN to open the key information page. Check the left navigation bar to see the specific encryption type. It might be an AWS managed key (aws/s3) or a customer-managed key.
```

Comment:
```suggestion
- If the AWS KMS key ARN in the area is displayed in blue with a link, click the key ARN to open the key information page. Check the left navigation bar to see the specific encryption type. It might be an AWS managed key (aws/s3) or a customer managed key.
```